.github/workflows/docker-pr-build.yml

```diff
@@ -8,21 +8,16 @@ on:
 jobs:
```

Comment:
tips: Maybe you should add the following lines at the beginning of the YAML to avoid unnecessary parallel launches:
```yaml
concurrency:
  group: ${{ github.workflow }}-${{ github.head_ref }}
  cancel-in-progress: true
```

Reply:
Good idea!

Reply:
Done